crypto/bio/bio_lib.c

```diff
@@ -784,3 +784,102 @@ void bio_cleanup(void)
     CRYPTO_THREAD_lock_free(bio_type_lock);
     bio_type_lock = NULL;
 }
+
+/* Internal variant of the below BIO_wait() not calling BIOerr() */
+static int bio_wait(BIO *bio, time_t max_time, unsigned long milliseconds)
```

Comment:
Why "unsigned long" for milliseconds rather  than just "unsigned int"? We try to avoid "long".

Reply:
An amount of milliseconds can in general be quite hight, but for `bio_wait()` and `BIO_wait()` no very large numbers are needed. So I've changed this as well.